unity-environment/Assets/ML-Agents/Examples/Banana/Scripts/BananaAgent.cs

```diff
@@ -152,6 +166,40 @@ public void MoveAgent(float[] act)
         gameObject.GetComponent<Renderer>().material.color = Color.black;
     }
 
+    public void Unfreeze()
```

Comment:
Unfreeze, freeze, poison, unpoison, satiate and unsatiate should be private if they're not needed externally